api/src/main/java/org/pmiops/workbench/cdr/CdrDbConfig.java

```diff
@@ -50,7 +51,15 @@
 
     private boolean finishedInitialization = false;
 
-    private final Long defaultCdrVersionId;
+    // Each access tier has a default version, but we don't know which tier is desired.
+    //
+    // This is because the default CDR Version is only returned when we have no knowledge about
+    // which CDR version is desired, and therefore also the access tier.  We have to choose one,
+    // and the Registered Tier is the safer choice.
+    //
+    // See also https://precisionmedicineinitiative.atlassian.net/browse/RW-6432
+
+    private final Long defaultRTCdrVersionId;
```

Comment:
nit: I don't think this variable needed to be renamed - it's technically still globally the default for the CdrDbConfig - it just happens to also be the registered tier. Consumers just care about which is the default. If you do want the rename, I would probably spell out RT since I don't think this is standard vernacular (yet)

Reply:
ok